packages/itwinui-css/src/button/button.scss

```diff
@@ -11,7 +11,7 @@
   justify-content: center;
   position: relative;
   border-radius: var(--iui-border-radius-1);
-  line-height: var(--iui-size-l);
+  line-height: 1.2;
```

Comment:
One more thing before we merge: have you gone through all the images and tried to verify if all the changes look good? Some of them could be undesirable.

Reply:
**[DONE]** Yes, I have re-gone through all the test image changes. Most undesirable changes I found before were the other components targeting `.iui-button-icon` and not targeting the inner svg. That's why I made changes like [L117](https://github.com/iTwin/iTwinUI/pull/788/files#diff-187108cb5bf3f6ce0ad8a514769689db86ffe4102ee775050d0b4c02dda54bedR117) or [L130](https://github.com/iTwin/iTwinUI/pull/788/files#diff-30ad83636b1219b4049da82968e8c6e9e8ca2a8fa13072e432ef4a67b7b0c11aR130).

The other change was adding a wrapper span on [L648](https://github.com/iTwin/iTwinUI/pull/788/files#diff-0562194b7c80cbcdb6d2d8b799a8b3839b3c3e5e67da4287cb5a495fc2f18a36R648) and [L719](https://github.com/iTwin/iTwinUI/pull/788/files#diff-0562194b7c80cbcdb6d2d8b799a8b3839b3c3e5e67da4287cb5a495fc2f18a36R719) for uniformity.

Another point just to mention is that `line-height: 1` caused `68` test image changes, while the current `line-height: 1.2` causes `90` test image changes. I guess this does not matter since like we discussed, we should focus on what looks better and not what causes the least test changes. Thus, I went with `line-height: 1`.